main/tests/server/src/com/google/refine/importers/ExcelImporterTests.java

```diff
@@ -85,11 +88,13 @@ public class ExcelImporterTests extends ImporterTest {
 
     private static final File xlsFileWithMultiSheets = createSheetsWithDifferentColumns(false);
     private static final File xlsxFileWithMultiSheets = createSheetsWithDifferentColumns(true);
+    private static final NumberFormat NUMBER_FORMAT = DecimalFormat.getInstance();
 
     @Override
     @BeforeTest
     public void init() {
         logger = LoggerFactory.getLogger(this.getClass());
+//        Locale.setDefault(Locale.US); // Some Excel formats are locale dependent, so pin our locale to en_US
```

Comment:
I think your use of `DecimalFormat` is better, so I would drop this comment. If we were setting the locale manually like this, I think we should also restore it at the end of the test.
Alternatively if we leave the comment in, I would add a comment to explain why it is commented out - otherwise I would find it confusing (especially because it's not necessarily obvious that the NumberFormat above is the solution to the locale dependency).

Reply:
Done